lib/models/events/event.js

```diff
@@ -0,0 +1,113 @@
+"use strict";
+
+/**
+ * Create a store event.
+ * @constructor
+ * @param {string} roomId The matrix room ID
+ * @param {string} eventId The matrix event ID
+ * @param {string} remoteRoomId The remote room ID
+ * @param {string} remoteEventId The remote event ID
+ * @param {any} extras Any extra data that may be included with the event.
+ */
+function StoreEvent(roomId, eventId, remoteRoomId, remoteEventId, extras) {
```

Comment:
Can we call this Store**d**Event please? It feels a bit like a typo when reading through the code, despite it being correct as a "store event".